File: soal3.py
def camelcase(s):
    count = 1  # Minimal ada 1 kata
    for char in s:
        if char.isupper():  # Setiap huruf kapital menandai kata baru
            count += 1
    return count

# Contoh penggunaan
if __name__ == "__main__":
    s = input("Masukkan string CamelCase: ").strip()
    print(camelcase(s))

File: soal3.ts
function camelcase(s: string): number {
    let count = 1; // Minimal ada 1 kata dalam CamelCase

    for (let char of s) {
        if (char >= 'A' && char <= 'Z') { // Cek huruf kapital
            count += 1;
        }
    }

    return count;
}

// Contoh penggunaan
const inputString: string = "saveChangesInTheEditor";
console.log(camelcase(inputString)); // Output: 5
